packaging/linux/common/khiops_env.in

```diff
@@ -79,11 +79,19 @@ if [ -f "/usr/share/khiops/khiops.jar" ]; then
     fi
 fi
 
+get_script_dir(){
+    SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+    echo "$SCRIPT_DIR/"
+}
+
 @ADDITIONAL_EN_VAR@
 
+KHIOPS_PATH=@KHIOPS_PATH@@MODL_NAME@
+KHIOPS_COCLUSTERING_PATH=@KHIOPS_COCLUSTERING_PATH@MODL_Coclustering
+
 # Number of processes in use (must be set according to the physical cores number)
 if [[ -z $KHIOPS_PROC_NUMBER ]]; then
-    KHIOPS_PROC_NUMBER=$(lscpu -b -p=Core,Socket | grep -v '^#' | sort -u | wc -l)
+    KHIOPS_PROC_NUMBER=$(@GET_PROC_NUMBER_PATH@_getprocnumber | head -n 1 )
```

Comment:
Why do we need `head -n 1` here?

Reply:
because in debug mode, the khiops framework add some info about allocation. To test khiops_env even in debug I've added this `head -n 1` . It is not done on windows because it is not very simple to implement and not very useful.

Reply:
Ok, I see. In this case, could one add a small comment just above, to explain this?